src/utop.mli

```diff
@@ -0,0 +1,15 @@
+open Import
+
+val exe_stanzas
+  : Jbuild.Stanza.t list
+  -> (Jbuild.Executables.t * Module.t String_map.t) option
+
+val add_module_rules
+  : Super_context.t
+  -> dir:Path.t
+  -> (unit, Lib.t list) Build.t
+  -> unit
+
+val src_dir : Path.t -> Path.t
+
+val target : Path.t -> Path.t
```

Comment:
Typing `Utop.utop_exe` feels a little unnatural to me. I might be the only one who thinks this way though.